controllers/machine_controller.go

```diff
@@ -184,6 +189,15 @@ func (r *MachineReconciler) reconcileDelete(ctx context.Context, cluster *cluste
 			return ctrl.Result{}, err
 		}
 	} else {
+		// Drain node before deletion
+		if _, exists := m.ObjectMeta.Annotations[clusterv1.ExcludeNodeDrainingAnnotation]; !exists {
+			klog.Infof("Draining node %q for machine %q", m.Status.NodeRef.Name, m.Name)
+			if err := r.drainNode(ctx, cluster, m.Status.NodeRef.Name, m.Name); err != nil {
+				r.recorder.Eventf(m, corev1.EventTypeWarning, "FailedDrainNode", "error draining Machine's node: %v", err)
+				return ctrl.Result{}, err
+			}
+			r.recorder.Eventf(m, corev1.EventTypeNormal, "SuccessfulDrainNode", "success draining Machine %q node %q", m.Name, m.Status.NodeRef.Name)
```

Comment:
```suggestion
			r.recorder.Eventf(m, corev1.EventTypeNormal, "SuccessfulDrainNode", "success draining Machine's node %q", m.Status.NodeRef.Name)
```

The event is against `m` (the machine), so we can omit the machine's name from the text.